### PR TITLE
Update some outdated requirements around docs/tests/dev (bug 846349)

### PR DESCRIPTION
**Pull request by diox**
https://bugzilla.mozilla.org/show_bug.cgi?id=846349

### CODE REVIEW

File: mkt/api/forms.py [hunk 1]
```diff
@@ -153,7 +153,10 @@ def __init__(self, *args, **kwargs):
 
     def __call__(self, value):
         super(URLValidator, self).__call__(value)
-        supports_scheme = lambda scheme: value.startswith(scheme + '://')
+
+        def supports_scheme(scheme):
+            return value.startswith(scheme + '://')
```
Comment (robhudson) [inline mkt/api/forms.py]: I always like `'%s://' % scheme` better

Comment (robhudson) [inline mkt/api/forms.py]: Or even `'{0}://'.format(scheme)` for future compat

File: requirements/test.txt [hunk 2]
```diff
@@ -7,11 +7,11 @@
...
 flake8==2.3.0
 pyflakes==0.8.1
-pep8==1.5.7
+pep8==1.6.2
 mccabe==0.3
 
 fudge==1.0.3
 mock==1.0.1
-psutil==0.2.0
+psutil==2.2.1
```
Comment (robhudson) [inline requirements/test.txt]: Is this used anywhere? I'm not seeing it imported but perhaps it's a dependency for something else. If so, should we document when a requirement is added but only as a dependency?

**Comment by robhudson** [comment]
r+wc

File: requirements/test.txt (hunk 2 above)
Comment (diox) [inline requirements/test.txt]: We should document that, but it's a pretty big task...

This particular package does not seem to be used anywhere. I ran a reverse dependency check and could not find a package requiring it. It was added in fe6836bc2ec68cc09fc5c2da63fca3596e13b047 (thanks, `git bisect`!) for qunit tests, which we no longer do. I'll remove it!

Comment (robhudson) [inline requirements/test.txt]: Yay, requirements--.

**Pull request merged**